### PR TITLE
TOLTECS: Fix warnings for subscript is above array bounds.

### PR DESCRIPTION
**Pull request by fedor4ever**
engines/toltecs/script.cpp:527:10: warning: array subscript is above array bounds [-Warray-bounds]
   warning("Getting unimplemented game variable %s (%d)", varNames[variable], variable);
Logging print garbage because array varNames[] has 22 elements.

### CODE REVIEW

**Comment by fedor4ever** [comment]
I wonder what compiler you use for build. Gcc print warning for that in January.

File: engines/toltecs/script.cpp [hunk 1]
```diff
@@ -498,7 +498,10 @@ void ScriptInterpreter::execScriptFunction(uint16 index) {
 }
 
 int16 ScriptInterpreter::getGameVar(uint variable) {
-	debug(2, "ScriptInterpreter::getGameVar(%d{%s})", variable, varNames[variable]);
+	if(variable > 21)
```
Comment (bluegr) [inline engines/toltecs/script.cpp:501]:
```suggestion
	if (variable > 21)
```

File: engines/toltecs/script.cpp [hunk 2]
```diff
@@ -524,13 +527,16 @@ int16 ScriptInterpreter::getGameVar(uint variable) {
...
 	default:
-		warning("Getting unimplemented game variable %s (%d)", varNames[variable], variable);
+		warning("Getting unimplemented game variable %d", variable);
 		return 0;
 	}
 }
 
 void ScriptInterpreter::setGameVar(uint variable, int16 value) {
-	debug(2, "ScriptInterpreter::setGameVar(%d{%s}, %d)", variable, varNames[variable], value);
+	if(variable > 21)
```
Comment (bluegr) [inline engines/toltecs/script.cpp:536]:
```suggestion
	if (variable > 21)
```

**Comment by bluegr** [comment]
Thanks! Squashing (to merge the whitespace fixes)

**Pull request merged**